crates/yttrium/src/smart_accounts/safe.rs

```diff
@@ -0,0 +1,115 @@
+use alloy::{
```

Comment:
I've moved this to `safe.rs` rather than `mod.rs` as I find it [easier find things and navigate Rust codebases](https://kerkour.com/rust-how-to-organize-large-workspaces#dont-put-any-code-in-your-modrs-and-librs-files)

Reply:
I actually don't like the module organization you are using. I find files by navigating in the file tree, not by searching for them. This current module structure has related files spread out.